ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/disconnected/provision/cucushift-installer-rehearse-gcp-ipi-disconnected-provision-chain.yaml

```diff
@@ -9,6 +9,7 @@ chain:
   - ref: ipi-conf-mirror
   - ref: ipi-conf-gcp
   - ref: ipi-install-install # using the step rather than "ipi-install" chain due to the first 2 steps
+  - ref: mirror-images-icsp-olm-operators
```

Comment:
Nothing is mirrored. Why we use this name 'mirror-images-icsp-olm-operators'?   I'd like some names like 'enable-qe-catalogsource' or something else

Reply:
The step naming depends on the folders, see: https://docs.ci.openshift.org/docs/architecture/step-registry/#step